bundle.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM scratch
```

Comment:
I would rather not, this is the file generated by the `make bundle` and will be autogenrated if not found whenever we run `make bundle` this will be genrated in the root of our dir

Reply:
oh ok ...